lib/ObjWriter/objwriter.cpp

```diff
@@ -420,6 +425,48 @@ extern "C" void EmitFrameInfo(ObjectWriter *OW, const char *FunctionName,
                 4);
 }
 
+extern "C" void EmitCFIStart(ObjectWriter *OW, int Offset) {
```

Comment:
Offset is not used, instead MC API will create/insert label to the code directly. But I'd like to keep this on purpose in case we need offset information later. The same thing applies to the following APIs.